com.prateekmedia.appimagepool.yaml

```
@@ -0,0 +1,50 @@
+app-id: com.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
```

```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

This is assuming the app supports Wayland.